docs/customization.rst

```diff
@@ -145,6 +145,23 @@ With this hook you can add additional test cases that will be executed in Hypoth
             Case(endpoint=context.endpoint, query={"foo": "bar"})
         )
 
+``add_case``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+For each add_case hook and for each endpoint, we create an additional, duplicate test case. We pass the Case object from the duplicate test to the add_case hook.
```

Comment:
Could you, please, wrap each `add_case` name in double-quotes so it is highlighted:

```
``add_case``
```